crypto/jwx.go

```diff
@@ -53,6 +53,30 @@ func (client *Crypto) SignJWT(claims map[string]interface{}, kid string) (token
 	return
 }
 
+func (client *Crypto) SignJWS(payload []byte, protectedHeaders map[string]interface{}, kid string) (string, error) {
+	headers := jws.NewHeaders()
```

Comment:
I noticed the newest version handles headers slightly different, adding protected headers was more difficult. Is there a test that checks for secured headers in the JWS?

Reply:
Yes there is an assertion for a protected header in the test